hazelcast/src/main/java/com/hazelcast/client/impl/operations/ClientReAuthOperation.java

```diff
@@ -55,7 +55,7 @@ public void run() throws Exception {
 
     @Override
     public boolean returnsResponse() {
-        return Boolean.TRUE;
+        return Boolean.FALSE;
```

Comment:
just nitpicking here, this is unnecessary unboxing, we could just return the primitive `false` directly.